src/store/modules/product/actions.ts

```diff
@@ -84,11 +84,32 @@ const actions: ActionTree<ProductState, RootState> = {
     commit(types.PRODUCT_ADD_TO_UPLD_PRDTS, { product: payload })
   },
 
-  updateCurrentProduct ({ commit, state }, payload) {
+  async updateCurrentProduct ({ commit, state }, payload) {
     // search in uploadProducts that if the clicked product is already in the upload list and set it as current product
-    const currentProduct = state.uploadProducts[payload.product.sku]
-    commit(types.PRODUCT_CURRENT_UPDATED, { product: currentProduct ? currentProduct : payload.product })
-  }
+    const currentProduct = state.uploadProducts[payload.currentSku]
+    // console.log(payload.currentSku)
+    if(currentProduct!==undefined){
```

Comment:
```suggestion
    if(currentProduct){
```

Reply:
Updated Sir.
Thank you sir for reviewing my code.